index.html

```diff
@@ -271,11 +271,12 @@
 		</span>
 		<div id="mainmenu" class="row" style="opacity: 0;">
 			
-			<div id="container-1" title="Add Group Chat to OBS"  alt="Add Group Chat to OBS" tabindex="2" role="button" aria-pressed="false" onkeyup="enterPressedClick(event,this);" class="column columnfade pointer rounded card" style=" overflow-y: auto;">
+			<div id="container-1" title="Add Group Chat to OBS"  alt="Add Group Chat to OBS" tabindex="2" role="button" aria-pressed="false" onkeyup="enterPressedClick(event,this);" class="column columnfade pointer rounded card">
```

Comment:
Using line awesome or other font packs for the main buttons doesn't work for me, since when you load the page the first time, you don't see the icons load for a second or two.

These buttons need to load quickly, and not wait for all of line awesome to load.